src/Modules/Scheduler.asm

```diff
@@ -242,6 +242,7 @@ scheduler_steps_odd_status_frame_max_loaded:
     clr  Flag_Demag_Notify
     clr  Flag_Desync_Notify
     clr  Flag_Stall_Notify
+    mov  Demag_Detected_Metric_Max, #120 ; Reset the Max metric so the next value would reflect maximum seen during the time in between STATUS telemetry packets
```

Comment:
Please reset to `#0`.

When building the status frame, we skip calculation if carry flag is set after subtraction of min value, thus skipping costly division (takes 4 cycles). If setting to min value, carry flag will not be set after subtraction, resulting in having to go through the division branch. Value wise the result will be the same though...

IMHO it also adds to readability not having the "magic" `#120` there.

Reply:
With this change can we use `clr` instead to match previous code?

Reply:
@hyp0dermik-code you can `clr` BITs and registers, the Flags are all BITs. `Demag_Detected_Metric_Max` is just a symbol holding the address of the data space it [has been assigned](https://github.com/bird-sanctuary/bluejay/blob/1b0802dbbf28f27572ba08b0a292db8adae29652/src/Bluejay.asm#L225).